.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.rs]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Hmm, it at least didn't delete the existing final newlines. Doesn't look like it has any option to control newline creation or not.